harvesting/esciencecenter.py

```diff
@@ -0,0 +1,469 @@
+    new_person_data["createdAt"] = get_current_time_string()
+    new_person_data["createdBy"] = "esciencecenter.nl.crawler"
+    new_person_data["updatedAt"] = new_person_data["createdAt"]
+    new_person_data["updatedBy"] = new_person_data["createdBy"]
+    return(new_person_data)
+
+
+def add_new_person_to_database(token,api_persons_data,name):
+    new_person_data = make_new_person_data(name)
+    person_api_url = "https://localhost/api/person"
+    person_api_response = requests.post(person_api_url,json=new_person_data,headers={'Authorization': 'Bearer %s' % token},verify=False)
+    if request_was_successful(person_api_response):
+        api_persons_data.append(new_person_data)
+        print(f'added new person "{name}"')
+        return(new_person_data)
+    else: 
+        logger.warning(f"failed storing person in database: {name}")
+
+
+def get_suffix(url):
+    url_parts = url.strip().split(".")
+    if len(url_parts) > 0: return(url_parts[-1])
+    else: return("")
+
+
+def make_new_organization_data(name_data):
+    new_organization_data = {"primaryKey":{"id":str(uuid.uuid4()),"collection":"organization"}}
+    new_organization_data["createdAt"] = get_current_time_string()
+    new_organization_data["createdBy"] = "esciencecenter.nl.crawler"
+    new_organization_data["updatedAt"] = new_organization_data["createdAt"]
+    new_organization_data["updatedBy"] = new_organization_data["createdBy"]
+    new_organization_data["url"] = "http://fix.me"
+    if "name" in name_data: new_organization_data["name"] = name_data["name"]
+    img_data = ""
+    img_suffix = "png"
+    if "img_url" in name_data:
+        img_response = requests.get(name_data["img_url"])
+        if request_was_successful(img_response): 
+            img_data = base64.b64encode(img_response.content).decode("utf-8") 
+            img_suffix = get_suffix(name_data["img_url"])
+    new_organization_data["logo"] = {"data":img_data, "mimeType":"image/"+img_suffix}
+    return(new_organization_data)
+
+
+def add_new_organization_to_database(token,api_organizations_data,name_data):
+    new_organization_data = make_new_organization_data(name_data)
+    organization_api_url = "https://localhost/api/organization"
+    organization_api_response = requests.post(organization_api_url,json=new_organization_data,headers={'Authorization': 'Bearer %s' % token},verify=False)
+    if request_was_successful(organization_api_response):
+        api_organizations_data.append(new_organization_data)
+        print(f'added new organization "{name_data["name"]}"')
+        return(new_organization_data)
+    else: 
+        logger.warning(f"failed storing organization in database: {name}")
+        
+
+# see required list and patterns in backend/schemas/project.json
+def make_new_project_data(project_data_source):
+    new_project_data = {}
+    new_project_data["callUrl"] = "https://doi.org/fix.me"
```

Comment:
```suggestion
    new_project_data["callUrl"] = "https://doi.org/FIXME"
```

(for the conditional to work correctly, see https://github.com/research-software-directory/research-software-directory/blob/5d3f8ba08927dd599f61751e75a6cfcdab1271c1/frontend/templates/project/sidebar.html#L1)

Reply:
Ok, updated.